.github/workflows/main.yml

```diff
@@ -90,9 +90,15 @@ jobs:
           path: ~/.hgexternalcache
           key: ${{ runner.os }}-${{ hashFiles('*/external/binaries-list', '*/*/external/binaries-list') }}
           restore-keys: ${{ runner.os }}-
+        
+      - name: Setup Gradle properties
+        run: |
+          mkdir -p ~/.gradle
+          echo "org.gradle.java.home=$JAVA_HOME_11_X64" >> ~/.gradle/gradle.properties
+          cat ~/.gradle/gradle.properties
```

Comment:
build is green.

gradle daemon didn't want to run on JDK 19 (`General error during conversion: Unsupported class file major version 63`)

looks like we can tell the gradle daemon which jdk to use. Similar like with mvnd.

Its interesting to me why this is working locally though. Since I never told it to use any JDK other than the one in JAVA_HOME.